recipes/libassert/all/conanfile.py

```diff
@@ -92,6 +102,17 @@ def package(self):
              src=self.source_folder)
         cmake = CMake(self)
         cmake.install()
+        
+        if Version(self.version) >= Version("1.2.1"):
```

Comment:
I think we could do this unconditionally, looks like the test_package also to avoid moving the dll's in the test_package